Readme.md

```diff
@@ -36,13 +36,15 @@ data = [[ 0.80,  0.55,  0.22,  0.03],
 
 ```python
 from minisom import MiniSom    
-som = MiniSom(6, 6, 4, sigma=0.3, learning_rate=0.5) # initialization of 6x6 SOM
+# initialization of 6x6 SOM with an input of dimension of 4 for training.
+# The initial sigma and learning rate are 0.3 and 0.5, respectively. They decline by dividing (1+t/T) where T is half num_iteration.
```

Comment:
No need to be specific here, sigma and learning rate are the name of the parameters.

Reply:
Sure. I keep these for my personal use only. :)